src/collections/members/aditi-ahuja/index.mdx

```diff
@@ -0,0 +1,14 @@
+---
+name: Aditi Ahuja
+position: Contributor
+image_path: ./aditi-ahuja.jpg
+github: metonymic-smokey
+twitter: Aditi09039216
+linkedin: aditi-ahuja-334394193
+location: Bangalore, India
+bio: Aditi Ahuja is a Student developer from Bangalore, India passionate about venturing into newer, niche fields.
+Open to challenges and opportunities either technical or otherwise
```

Comment:
Try adding a space before this line, and build. See if that works.

Reply:
It didn't work

Reply:
This should be one line with spaces, not separate lines. And, don't add an empty line after it.